components/admin_console/group_settings/group_details/group_profile_and_settings.jsx

```diff
@@ -76,7 +77,7 @@ export const GroupProfileAndSettings = ({displayname, mentionname, allowReferenc
                 name={mentionname}
                 title={t('admin.group_settings.group_details.group_mention.name')}
                 titleDefault={'Group Mention:'}
-                customID={'GroupMention'}
+                customID={'group_mention'}
```

Comment:
these changes are also seemingly unrelated but again it made sense to set these ids as snake case since the rest of the ids in this file also use the same format and these two components were the outliers